agent/wpthook/test_state.cc

```diff
@@ -306,7 +306,7 @@ void TestState::OnLoad() {
     QueryPerformanceCounter(&_on_load);
     GetCPUTime(_doc_cpu_time, _doc_total_time);
     ActivityDetected();
-    if (!_test._useHawk) {
+    if (!_test._useHawk && !_test._noScreenshots) {

```

Comment:
we could simplify the code by:
 - setting `_noScreenshots` to true when we want to use Hawk and only testing for `_noScreenshots`
 - Running image tools `if (!test._noScreenshots || test._useHawk)`